src/v/redpanda/tests/fixture.h

```diff
@@ -63,8 +64,8 @@ class redpanda_thread_fixture {
       std::optional<scheduling_groups> sch_groups,
       bool remove_on_shutdown)
       : app(ssx::sformat("redpanda-{}", node_id()))
-      , proxy_port(proxy_port)
-      , schema_reg_port(schema_reg_port)
+      , proxy_ep{.address = net::unresolved_address(host_name, proxy_port)}
+      , schema_reg_ep{.address = net::unresolved_address(host_name, schema_reg_port)}
```

Comment:
The only change necessary for this commit is `broker_endpoint` -> `rest_authn_endpoint`
The changes to the ports should go in the unit test commits.